airbyte-integrations/connectors/source-hubspot/acceptance-test-config.yml

```diff
@@ -43,6 +43,8 @@ acceptance_tests:
         companies:
         - name: properties/hs_time_*
           bypass_reason: Hubspot time depend on current time
+        - name: properties/hs_all_*
+          bypass_reason: This property is floating from sync to sync
```

Comment:
This is ignoring all `hs_all_whatever` fields? I find it hard to believe that all of them float if the data's not changing. E.g. these ones are probably stable: `"hs_all_accessible_team_ids": null, "hs_all_owner_ids": "52550153", "hs_all_team_ids": null,` Are there specific ones that are causing issues?

Reply:
I don't understand why they could change though 😅 there won't be new owners or teams, etc. But now I'm also confused, as there are still `hs_all` fields in the expeceted records, so maybe this isn't ignoring what I think it is?

Reply:
Ok, I'll remove this from `acceptance-test-config.yaml`

Reply:
Just for posterity's sake, looking at the diff, I see some completely new `hs_all` fields, which is annoying when hubspot changes stuff, but I think its more of "hubspot is giving us new fields" and not that the data within that field is changing:

![image](https://user-images.githubusercontent.com/22333512/234905592-7a817409-ae00-4a3e-9ef9-3600dde4dbd2.png)

I think it makes sense to update the expected records to contain the new fields in that case.